ansible/influxdb-meta/tests/vagrant.yml

```diff
@@ -0,0 +1,13 @@
+---
+- hosts: all
+  pre_tasks:
+    - name: "Populate hosts file"
```

Comment:
This works on Ubuntu, but fails on CentOS. This is what I had to use get it to work on CentOS:

```
+    - name: Populate hosts file
+      lineinfile:
+        dest: /etc/hosts
+        regexp: '.*{{ item }}$'
+        line: "{{ hostvars[item]['ansible_all_ipv4_addresses']|last }}  {{item}}"
+        state: present
+      when: >
+        groups['meta-nodes'] is defined
+        and
+        ansible_os_family == "RedHat"
+      with_items: "{{ groups['meta-nodes'] }}"
```

We might want to split this logic into two sections, one for Ubuntu/Debian and another for CentOS/RHEL.